src/main/java/org/opencds/cqf/providers/CqlExecutionProvider.java

```diff
@@ -211,7 +211,14 @@ public Bundle evaluate(@OperationParam(name = "code") String code,
             @OperationParam(name = "terminologyServiceUri") String terminologyServiceUri,
             @OperationParam(name = "terminologyUser") String terminologyUser,
             @OperationParam(name = "terminologyPass") String terminologyPass,
+            @OperationParam(name = "startingContext") String startingContext,
```

Comment:
I think this would make more sense as just "context"

Reply:
Will make that change